src/plone/app/transmogrifier/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""Functions utils."""
+from Products.CMFPlone.utils import safe_unicode
```

Comment:
`safe_unicode` is deprecated - use `safe_text`.

Reply:
@jensens I changed to safe_text.

Reply:
@jensens `Plone < 5.2` doesn't have `safe_text`. See:

https://github.com/collective/plone.app.transmogrifier/runs/3196672035?check_suite_focus=true#step:9:819

 I went back to `safe_unicode`

Reply:
Indeed, please conditional import it.

Reply:
Done.